modules/adomikAnalyticsAdapter.js

```diff
@@ -1,8 +1,7 @@
 import adapter from 'src/AnalyticsAdapter';
 import CONSTANTS from 'src/constants.json';
 import adaptermanager from 'src/adaptermanager';
-import find from 'core-js/library/fn/array/find';
-import findIndex from 'core-js/library/fn/array/find-index';
```

Comment:
These functions must be polyfilled to run in IE.  You can remove the polyfills if you'd like, but you'd have to remove the usages of `find` and `findIndex` as well, otherwise our browserstack tests will fail.

https://github.com/prebid/Prebid.js/pull/1918

Reply:
ok, done

Reply:
@snapwich Is it good for you ?

Reply:
@snapwich Could you take a look at this please ?